src/locales/locale-en.json

```diff
@@ -307,6 +307,7 @@
     "SERVER_ERROR": "Server error"
   },
   "GROUP": {
+    "ACTIVE": "Active",
```

Comment:
This translation has been added, but seems unused?

Reply:
True, it's a leftover. Thanks for pointing it out!